src/utils/index.ts

```diff
@@ -2,7 +2,7 @@
 import type { AxiosRequestHeaders } from 'axios';
 
 // config
-const { rideToWorkByBikeConfig } = await import('../boot/global_vars');
+import rideToWorkByBikeConfig from '../boot/global_vars';
```

Comment:
@maceksimon this change does not work as expected `rideToWorkByBikeConfig.apiVersion` var value is `undefined` (during real  web browser register process testing).

Reply:
Ok, however, with this change tests fail because the top level await is not supported in the target browser set defined in quasar.config.

Reply:
Tests are ok, but build process of app fail, because old browsers doesn't support this new feature. I will try solve it.

Reply:
Fixed with https://github.com/auto-mat/ride-to-work-by-bike-frontend/pull/608/commits/88e57c4003a91b5ae759a6f2ec495ec61bacb838.